src/SAWScript/Prover/What4.hs

```diff
@@ -44,12 +57,27 @@ proveWhat4_sym ::
   Prop ->
   IO (Maybe [(String, FirstOrderValue)], SolverStats)
 proveWhat4_sym solver un sc hashConsing t =
-  do -- TODO: get rid of GlobalNonceGenerator ???
-     sym <- B.newExprBuilder B.FloatRealRepr St globalNonceGenerator
-     cacheTermsSetting <- getOptionSetting B.cacheTerms $ getConfiguration sym
-     _ <- setOpt cacheTermsSetting hashConsing
+  do sym <- setupWhat4_sym hashConsing
      proveWhat4_solver solver sym un sc t
 
+proveExportWhat4_sym ::
+  SolverAdapter St ->
+  [String] ->
+  SharedContext ->
+  Bool ->
+  FilePath ->
+  Prop ->
+  IO (Maybe [(String, FirstOrderValue)], SolverStats)
+proveExportWhat4_sym solver un sc hashConsing outFilePath t =
+  do sym <- setupWhat4_sym hashConsing
+
+     -- Write smt out
+     (_, _, lit, stats) <- setupWhat4_solver solver sym un sc t
+     withFile outFilePath AppendMode $ \handle ->
```

Comment:
`AppendMode` -> `WriteMode`, otherwise the running the same command multiple times would result in the same content being appended over and over again

Reply:
Ah, I had it set to `AppendMode` for debugging and forgot to switch it back!  Good catch.  Fixed.